### PR TITLE
[AC-5050] - Add prerequisites and installation instructions for the new deployment and build system

### PR DESCRIPTION
**Pull request by yotammanor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,227 +3,144 @@ _Copyright (c) 2017 MassChallenge, Inc._
...
+`make run-server`
+or
+`make run-server-1`
+or
+`make run-server debug=1`
+
+* Note that exiting run-server (via ctrl+c) will **not** kill
+the server, but instead it will return to detached mode, and
+run in the background.
+To stop to server, run `make stop-server` as well.
```
Comment (pbx) [inline README.md]: "To stop the server"

File: README.md [hunk 2]
```diff
@@ -3,227 +3,144 @@ _Copyright (c) 2017 MassChallenge, Inc._
...
+At times, you may want to see the gunicorn + supervisor
+server as is being run in production. This mode also shows
+a more detailed output of the docker-compose up command 
+that is executed in the background.
 
-### Add New Strings for Translation
+This can be done by running
+`make run-server-0`
+or
+`make run-serer debug=0`
```
Comment (pbx) [inline README.md]: s/serer/server/

File: README.md [hunk 3]
```diff
@@ -235,97 +152,73 @@ To give a user access to the v1 api, issue the command with
...
-
-Install `ecs` (version 1.4.3 or later): `pip install ecs-deploy`
-
-You will also need to configure an AWS access key and secret key for
-doing the deploy.  Steps for creating access keys can be found here:
-http://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey
+### Release and Deploy
 
-At the end of this process you should know both your access key and secret key.
+Impact-api is configured such that every successful travis build
```
Comment (pbx) [inline README.md]: Cap "Travis"

**Review by pbx — CHANGES_REQUESTED** [review]
Good stuff, thanks again. My notes are mostly copy edits.

File: README.md [hunk 4]
```diff
@@ -3,227 +3,144 @@ _Copyright (c) 2017 MassChallenge, Inc._
...
-    make superuser
+This section describes the more common commands you may,
+need when working on this platform. For a full list of the
+commands, refer to `make help`.
 
 ### Rebuilding the container
 
-This should be done whenever the requirements change or `INSTALLED_APPS`
-is changed.
+This should be done whenever the requirements change.
```
[the remark this answers is not in the document]
Comment (yotammanor) [reply README.md]: added a more elaborate example.

File: README.md [hunk 5]
```diff
@@ -3,227 +3,144 @@ _Copyright (c) 2017 MassChallenge, Inc._
...
-Note that this command will reset the DB, so any changes you've made
-before (i.e. `make dbload`, `make superuser`...) should be
-re-executed.
-
-
-### Creating a release:
-
-To create a release run the ```make release``` command which will tag your current HEAD
-with a semantic release tag and also tag the related django-accelerator and accelerate repos.
+* Note that these commands will delete the DB, so any
```
[the remark this answers is not in the document]
Comment (yotammanor) [reply README.md]: done

File: README.md [hunk 6]
```diff
@@ -3,227 +3,144 @@ _Copyright (c) 2017 MassChallenge, Inc._
...
-You can also invoke a debugger inline by adding the following line in
-your code:
-
-   import pdb;pdb.set_trace()
-
-Visiting a view that invokes the debug line will start a pdb prompt.
+To run an interactive instance of Django, run:
+`make run-server`
+or
+`make run-server-1`
```
[the remark this answers is not in the document]
Comment (yotammanor) [reply README.md]: They're all equivalent. Added that statement explicitly.

**Comment by yotammanor** [comment]
@pbx addressed your comments.

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (pbx) [reply README.md]: If they're equivalent, why do we introduce the alternative form here? Is there a downside to simply using `make run-server` as the prescribed command here?

**Review by pbx — CHANGES_REQUESTED** [review]
Great changes, I just have one remaining point of confusion.

**Comment by yotammanor** [comment]
@pbx re your last question - I was seeing it as an opportunity to introduce the available synonyms. They are all referring to the same implementation.

**Comment by pbx** [comment]
@yotammanor - another way to put my question is, what benefit does the developer reading this documentation gain from learning that `make run-server-1` is effectively an alias for `make run-server`? It feels like we are surfacing a Makefile implementation detail. This is not a big point by any means, I'm just focusing on making the experience of learning our tooling as smooth as possible for engineers new to it. 

If you feel like showing both names for this command is really the right thing to do, I don't need to die on this hill. I could live with it being merged as-is.

**Comment by yotammanor** [comment]
@pbx clarified that the other commands are aliases.

**Review by pbx — APPROVED** [review]

**Pull request merged**